fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/ReindexManager.java

```diff
@@ -78,7 +78,15 @@ public ReindexManager(final Stream<String> ids, final ReindexService reindexServ
         workers = new ArrayList<>();
         completedCount = new AtomicInteger(0);
         errorCount = new AtomicInteger(0);
-        for (var foo = 0; foo < config.getReindexingThreads(); foo += 1) {
+
+        final var workerCount = config.getReindexingThreads();
+
+        if (workerCount < 1) {
```

Comment:
@bbpennel Which block? Line 409? That block is called if someone specifies the threads manually

Reply:
I personally like that assert. It enforce a pre-condition of the class that is independent of the configuration source.

Reply:
@whikloj is correct, that's what I meant. Like I said, its not causing any harm, it's just double checking something that doesn't seem like it can ever happen presently. If its intent is to be a safety check in case there's ever a different configuration being provided here, that's fine, I was just pointing out the duplication in case it was unintentional.

Reply:
Yes, it's intentional, additional safetly. This was a really hard bug to find, and I don't want it to happen again. That class does not work with thread values less than 1, so it makes sense for it to enforce that condition.